Linux/sgx/libsgx_tsgxssl/tunistd.cpp

```diff
@@ -71,19 +71,19 @@ size_t sgxssl_write (int fd, const void *buf, size_t n)
 	// It is unreachable under the assumption that TLS support is not required.
```

Comment:
Not sure if this comment is still necessary. I think the API can be used not only in TLS.

Reply:
the comment is not unnecessary and removed. standard lib C APIs for file open/close/read/write.